src/render/blog/adapt-service-without-webapi.html.md

```diff
@@ -0,0 +1,505 @@
+which selects all records from the `authors` table.
+
+## Prepare the template
+
+The last thing left for us to build is a template configuration. We want the template to display data from `books-listing` and
+`authors-listing` services. This can be achieved by creating a couple of simple [Handlebars](https://github.com/Cognifide/knotx/wiki/HandlebarsKnot) templates in `books.html`:
+
+```html
+    <script data-knotx-knots="services,handlebars"
+            data-knotx-service="books-listing"
+            type="text/knotx-snippet">
+            {{#each _result}}
+              <div class="col-sm-4">
+                <div class="card">
+                  <div class="card-block">
+                    <h2 class="card-title">{{this.TITLE}}</h2>
+                    <h4 class="card-title">{{this.ISBN}}</h4>
+                    <p class="card-text">
+                      {{this.SYNOPSIS}}
+                    </p>
+                  </div>
+                </div>
+              </div>
+            {{/each}}
+    </script>
+```
+This tells _Knot.x_ to call the `books-listing` service and make the data available in the `_result` scope.
+We iterate over `_result` since it is a list of all books fetched from the database.
+
+```html
+    <script data-knotx-knots="services,handlebars"
+            data-knotx-service="authors-listing"
+            type="text/knotx-snippet">
+            {{#each _result}}
+              <div class="col-sm-4">
+                <div class="card">
+                  <div class="card-block">
+                    <h2 class="card-title">{{this.NAME}}</h2>
+                    <h4 class="card-title">{{this.AGE}}</h4>
+                  </div>
+                </div>
+              </div>
+            {{/each}}
+    </script>
+```
+
+This makes _Knot.x_ call the `authors-listing` service and expose the data in the `_result` scope.
+We iterate over the entries in `_result` since it is a list of all authors fetched from the database.
+
+The final markup of the template can be downloaded from our [GitHub repository for this tutorial](https://github.com/Knotx/knotx-tutorials/tree/master/adapt-service-without-webapi/content/local/books.html).
+
+# Run the example
+
+Now we have all the parts ready and can run the demo.
+The application directory should now contain the following artifacts:
+
+```
+├── knotx-standalone-1.0.0.json
+├── knotx-standalone-1.0.0.logback.xml
+├── app
+│   ├── adapt-service-without-webapi-1.0.0-fat.jar
+│   ├── knotx-standalone-1.0.0.fat.jar
+├── content
+│   ├── local
+│       ├── books.html
+```
+
+You can run the _Knot.x_ instance using the following command:
+
+`java -Dlogback.configurationFile=knotx-standalone-1.0.0.logback.xml -cp "app/*" io.knotx.launcher.LogbackLauncher -conf knotx-standalone-1.0.0.json`
+
+When you visit the page [http://localhost:8092/content/local/books.html](http://localhost:8092/content/local/books.html),
+ you will see books and authors from the database listed.
```

Comment:
You can mention here some options to play. E.g. add new entries to the DB and refresh the page, so new books or authors will be visible immediately.